fsw/public_inc/sadb_routine.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021, by the California Institute of Technology.
+ * ALL RIGHTS RESERVED. United States Government Sponsorship acknowledged.
+ * Any commercial use must be negotiated with the Office of Technology
+ * Transfer at the California Institute of Technology.
+ *
+ * This software may be subject to U.S. export control laws. By accepting
+ * this software, the user agrees to comply with all applicable U.S.
+ * export laws and regulations. User has the responsibility to obtain
+ * export licenses, or other export authority as may be required before
+ * exporting such information to foreign countries or providing access to
+ * foreign persons.
+ */
+
+#ifndef CRYPTOLIB_SADB_ROUTINE_H
+#define CRYPTOLIB_SADB_ROUTINE_H
+
+#ifdef NOS3 //NOS3/cFS build is ready
+#include "common_types.h"
+#include "osapi.h"
+#else //Assume build outside of NOS3/cFS infrastructure
```

Comment:
Good assumption that the build is outside nos3/cFS by default.  We're still having discussions about how to ensure that we maintain the spacecraft build/functionality.  Right now, we're leaning toward CryptoLib being a generic lib, and we will go back and adapt for spacecraft.  (This is a role reversal for how CryptoLib was first designed.)